### ISSUE TITLE
Docker installation process

### ISSUE THREAD

**Issue by gekid83** [cyborgbackup/cyborgbackup#55]
Hi,
i can't complete the docker installation process.
I receive this error: "No such service: web" after i tape "docker-compose exec web /bin/bash".

Regards,
Giacomo

**Comment by GaetanF** [comment]
Hello,

The container have been renamed since few commits. The new name of the container is "api".

Regards

**Comment by Jakethesnkae** [comment]
Hey there how would the command be done little confusing

**Comment by olitee** [comment]
Yeah, there's no bash shell - but you can achieve the same using **/bin/sh**

So `docker-compose exec api /bin/sh` should work.
